doc/tutorial/alignment_file/index.md

```diff
@@ -0,0 +1,309 @@
+# Alignment Input and Output in SeqAn3 {#tutorial_alignment_file}
+
+***Learning Objective:***
+
+<b>Learning Objective:</b> <br/>
+You will get an overview of how to read and write alignment files.
+This tutorial is a walk-through with links into the API documentation and also meant as a source for copy-and-paste code.
+
+\tutorial_head{Medium, 60 min, \ref setup\, \ref tutorial_alphabets\, \ref tutorial_sequence_file,}
+
+[TOC]
+
+# Introduction
+
+Alignment files are used to store pairwise alignments between two (biological) sequences.
+Common file formats are the Sequence Alignment/Map format (SAM) and BLAST output format.
+Next to the alignment, those formats store additional information like the start positions or mapping qualities.
+Alignment files are a little more complex than sequence files but the basic design is the same.
+If you are new to SeqAn3, we strongly recommend to do the [Sequence File Tutorial](#tutorial_alignment_file) first.
+
```

Comment:
Maybe give a link to the spec?

Reply:
I added a link to the spec in the SAM format section